client/src/__tests__/App.test.tsx

```diff
@@ -0,0 +1,16 @@
+import { act, render } from '@testing-library/react';
+import 'mobx-react-lite/batchingForReactDom';
+import React from 'react';
+import api from '~/api/test';
```

Comment:
Its better to have it inside `~/api/__mocks__`:
https://jestjs.io/docs/en/manual-mocks

Lets try to move it there